web/src/components/molecules/Asset/InfoBox/index.tsx

```diff
@@ -58,25 +57,34 @@ const InfoBoxWrapper = styled.div`
   max-width: 500px;
   max-height: 800px;
   text-align: left;
-  overflow: scroll;
   background-color: ${({ color }) => color};
+  border-radius: 6px;
 `;
 
 const Header = styled.div`
+  height: 50px;
   display: flex;
   justify-content: space-between;
   align-items: center;
+  padding: 8px 12px;
+  box-shadow: inset 0px -1px 0px #f0f0f0;
 `;
 
 const Title = styled.div`
```

Comment:
@nourbalaha , I think you can remove the property from the:
` <Title color={theme.base01}>{title}</Title>`
color is no longer used here.

Reply:
Makes sense, thanks!